SPECS-EXTENDED/libwpg/libwpg.spec

```
@@ -0,0 +1,229 @@
+Summary:        A library for import of WordPerfect Graphics images
+Name:           libwpg
+Version:        0.3.3
+Release:        9%{?dist}
+License:        LGPL-2.0-or-later OR MPL-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libwpg.sourceforge.net/
+Source0:        http://download.sourceforge.net/libwpg/%{name}-%{version}.tar.xz
+%global         apiversion 0.3
```

We're trying to keep the macros/globals/defines at the top to make them easier to find. And yes, the linter is still stubbornly moving them around.:)

```suggestion
%global         apiversion 0.3

Summary:        A library for import of WordPerfect Graphics images
Name:           libwpg
Version:        0.3.3
Release:        9%{?dist}
License:        LGPL-2.0-or-later OR MPL-2.0
Vendor:         Microsoft Corporation
Distribution:   Mariner
URL:            https://libwpg.sourceforge.net/
Source0:        http://download.sourceforge.net/libwpg/%{name}-%{version}.tar.xz
```